frontend/src/component/feature/FeatureView/FeatureViewHeader.tsx

```diff
@@ -174,10 +189,85 @@ const useLegacyVariants = (environments: IFeatureToggle['environments']) => {
     return enableLegacyVariants || existingLegacyVariantsExist;
 };
 
+type HeaderActionsProps = {
+    feature: IFeatureToggle;
+    showOnNarrowScreens?: boolean;
+    onFavorite: () => void;
+    handleCopyToClipboard: () => void;
+    isFeatureNameCopied: boolean;
+    openStaleDialog: () => void;
+    openDeleteDialog: () => void;
+};
+
+const HeaderActionsComponent = ({
```

Comment:
Moves the action into a separate component so that they can be rendered multiple times